antispam_bee.php

```diff
@@ -2193,10 +2193,14 @@ private static function _is_mobile() {
 	/**
 	 * Testing if we are on an AMP site.
 	 *
+	 * Starting with v2.0, amp_is_request() is the preferred method to check,
+	 * but we fall back to the then deprecated is_amp_endpoint() as needed.
+	 *
 	 * @return bool
 	 */
 	private static function _is_amp() {
-		return function_exists('is_amp_endpoint') && is_amp_endpoint();
+		return ( function_exists( 'amp_is_request' ) && amp_is_request() )
+		       || ( function_exists('is_amp_endpoint') && is_amp_endpoint() );
```

Comment:
There some space missing. `function_exists('is_amp_endpoint')` needs to be `function_exists( 'is_amp_endpoint' )`

This is why the test is failing.

Reply:
:+1: fixed.